src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  position: relative;
+  width: 100%;
+  margin-bottom: 20px;
+  display: block;
+}
+
+.icon {
+  position: absolute;
+  display: block;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon-SMALL {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  font-family: "Avenir", sans-serif;
+  color: #3e4e61;
+  font-weight: 300;
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg,  #fff 0%, #e1e7ed 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.input::placeholder,
+.input-SMALL::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.input-SMALL {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  box-sizing: border-box;
+  padding-left: 33px ;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  line-height: 19px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.

Reply:
not fixed

make sure you pushed your changes